### DISCUSSION TITLE
Suddenly, Title returning null in PnP Search Results

### DISCUSSION THREAD

**Discussion by MrWorkingCat** [microsoft-search/pnp-modern-search#2830] [General]
I am using version 4.5.4.

Could this be a result of a configuration change in our tenant, or are other users having the same issue.

See below an extract of the debug data:

```
.
.
.
"FileType": "doc",
.
.
.
"Path": "https://i---d.sharepoint.com/sites/NA---e/Active Document Library/1n3y01_.DOC",
.
.
.
"SiteTitle": "NA------de",
"SPSiteUrl": "https://i---d.sharepoint.com/sites/NA---e",
"Title": null,
"UniqueID": "{eb27935a-c632-5ffb-985c-3fe1bff6b485}",
.
.
.
```

The screenshot below shows more details.

![image](https://user-images.githubusercontent.com/4206482/223886410-a6506c97-0714-4d15-b484-c949da3d5120.png)

The file has a title different to null.

**Comment by wobba** [comment]
If you get null and expect to get a title, that is an API issue you have to log a support issue for. If logging support cases the easiest is to set up repro with classic search web parts, and report the issue for that setup.

**Reply by MrWorkingCat** [reply]
Thank you, great advice, will do.
